www/js/lib/zimfile.js

```diff
@@ -105,7 +105,6 @@ var readInt = function (data, offset, size) {
  * @property {Integer} mimeListPos Position of the MIME type list (also header size)
  * @property {Integer} mainPage Main page or 0xffffffff if no main page
  * @property {Integer} layoutPage Layout page or 0xffffffffff if no layout page
- * @property {String} zimType Extended property: currently either 'open' for OpenZIM file type, or 'zimit' for the warc2zim file type used by Zimit (set in zimArchive.js)
```

Comment:
This has been moved to the higher level (property of ZIMArchive instead of ZIMFile), as I feel it more appropriately belongs with the metadata. It describes the Archive type, not the File type.